frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "easymde": "^2.20.0",
     "font-awesome": "^4.7.0",
     "image-type": "^6.0.0",
+    "marked": "^4.3.0",
```

Comment:
```suggestion
```
This isn't actually needed anymore since I added marked in "peerDependencies" below.